docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.9"
+services:
+  dynamodb-local:
+    image: amazon/dynamodb-local:latest
+    ports:
+      - "18079:8000"
+    command: ["-jar", "DynamoDBLocal.jar", "-sharedDb", "-inMemory"]
+  dynamoInit:
+    depends_on:
+      - dynamodb-local
+    image: banst/awscli
+    environment:
+      AWS_ACCESS_KEY_ID: 'DUMMYIDEXAMPLE'
+      AWS_SECRET_ACCESS_KEY: 'DUMMYEXAMPLEKEY'
+    command:
+      dynamodb describe-limits --endpoint-url http://dynamodb-local:8000 --region us-west-2
```

Comment:
Will this (in the follow-up PRs) allow us to create the dynamo tables?

Reply:
Yes! the aws banst cli gives us access to all of the CLI commands for dynamo, including [create table](https://docs.aws.amazon.com/cli/latest/reference/dynamodb/create-table.html)